forest/daemon/src/daemon.rs

```diff
@@ -388,7 +390,15 @@ async fn maybe_fetch_snapshot(
 ) -> anyhow::Result<Config> {
     if should_fetch_snapshot {
         let snapshot_path = default_snapshot_dir(&config);
-        let path = snapshot_fetch(&snapshot_path, &config, &None, is_aria2_installed()).await?;
+        let path = retry!(
+            snapshot_fetch,
+            DEFAULT_RETRIES,
+            DEFAULT_DELAY,
```

Comment:
How about putting those into the daemon configuration? They don't have to be exposed as CLI flags, but it'd be nice to alter them via config file.

Reply:
fixed